<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParameterEstimationDialog</class>
 <widget class="QDialog" name="ParameterEstimationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ParameterEstimationDialog</string>
  </property>
  <widget class="QLineEdit" name="lineEditTestDataDirectory">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Image directory:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonTestDataDirectory">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Final parsing output:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditOutputDirectory">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOutputDirectory">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>100</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSegmentationOutputDirectory">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Segmentation output:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSegmentationOutputDirectory">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonInitialOutputDirectory">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>70</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Initial parsing output:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditInitialOutputDirectory">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditTestDataDirectory</tabstop>
  <tabstop>pushButtonOK</tabstop>
  <tabstop>pushButtonCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
